dev/provider_packages/prepare_provider_packages.py

```diff
@@ -438,7 +445,7 @@ def is_from_the_expected_base_package(the_class: Type, expected_package: str) ->
     return the_class.__module__.startswith(expected_package)
 
 
-def inherits_from(the_class: Type, expected_ancestor: Type) -> bool:
+def inherits_from(the_class: Type, expected_ancestor: Optional[Type]) -> bool:
```

Comment:
should we have the default value as `None` too since it is `Optional`

Reply:
It's not strictly necessary. The 'optional' does not imply default (the case here was that the mypy realised that we can both - expect None here and pass it. but it does not have to be default (I added it thought).